bin/common.bash

```diff
@@ -643,3 +643,22 @@ with_s3cfg() {
   #       bucket needs to be created.
   sops_exec_file_no_fifo "${s3cfg}" 'S3COMMAND_CONFIG_FILE="{}" '"${*}"
 }
+
+check_node_label() {
+  local cluster="${1}"
+  local label="${2}"
+
+  local -a nodes_missing_node_group_label
+
+  readarray -t nodes_missing_node_group_label <<<"$(with_kubeconfig "${config["kube_config_${cluster}"]}" kubectl get no -l "!${label}" -o yaml | yq '.items[] | .metadata.name')"
```

Comment:
```suggestion
  readarray -t nodes_missing_node_group_label <<<"$(with_kubeconfig "${config["kube_config_${cluster}"]}" kubectl get node -l "!${label}" -o yaml | yq '.items[] | .metadata.name')"
```

I'd prefer readability over saving two characters in a script :)

Thought: `kubectl get node -o name|cut -d/ -f2-` ?

Reply:
Both good suggestions! :) https://github.com/elastisys/compliantkubernetes-apps/pull/2535/commits/32e55b1be7820723264b53dab52f810b893fbf9b